src/main/resources/application.yaml

```diff
@@ -186,3 +188,12 @@ springdoc:
     enabled: true
     operationsSorter: method
   writer-with-order-by-keys: true
+
+professional:
+  refdata:
+      pageSize: ${PROFESSIONAL_REFDATA_PAGE_SIZE:100}
+  role:
+    mapping:
+      scheduling:
+        enabled: ${PROFESSIONAL_SCHEDULING_ENABLED:false}
+        cron: ${PROFESSIONAL_ROLE_MAPPING_CRON:"*/30 * * * * *"}
```

Comment:
Dan and Suman have just determined that we will have multiple schedules in crons, so we probably want this value to be specific per Process

Reply:
updated in [9375d38](https://github.com/hmcts/am-org-role-mapping-service/pull/1743/commits/9375d389ba6ce2749085574411c4ea75aa7fd2f6)